syntax = "proto3";
package RouteServer;

import "BaseDefine.proto";

message UserInfo {
    BaseDefine.ClientType client_type = 1;
    string msg_server_name = 2;
}